readme.md

```diff
@@ -128,41 +128,67 @@ This is a one-time pay for each task. Payments will be in XMR.
 
 We can negotiate payment if you feel the bounty price is too low.
 ```
-$2000 - implement peer discovery system that connects to other peers that have synced the full copy of the database
-files to work with: src/daemon.cpp, src/server.cpp, src/client.cpp
-libraries used: System network libraries (Linux Sockets, Winsock2, etc.)
-difficulty: hard
-job status: not filled
-
-$1000 - create a search engine that can search for item names, seller names and products, etc.
-
-$1000 - fix and optimize graphics and add graphical user interface features (such as scrollbar and scrollarea)
-files to work with: external/dokun-ui/src/renderer.cpp; external/dokun-ui/include/renderer.hpp
-libraries used: opengl, vulkan (optional)
-difficulty: intermediate
-job status: not filled
-
-$500 - convert and export qr code data to PNG
-files to work with: src/qr.cpp; include/qr.hpp
-libraries used: libpng, QR Code generator
-difficulty: intermediate
-job status: not filled
-
-$250 - fix decoding error in functions using EVP_PKEY_encrypt() and EVP_PKEY_decrypt()
-files to work with: src/encryptor.cpp; include/encryptor.hpp
-libraries used: openssl
-difficulty: intermediate
-job status: not filled
-
-$250 - fetch a list of monero nodes
-libraries used: ?
-difficulty: intermediate
-job status: not filled
-
-$100 - create an organized documentation on the neroshop codebase including functions, classes, etc.
-tools to use: doxygen or any other tools
-difficulty: easy
-job status: not filled
+Details: Implement peer discovery system that broadcasts/multicasts messages to other peers, connects to other peers, stores their IP addresses and shares it with other neroshop nodes.
+Bounty Reward: $1000-$3000
+Files: src/daemon.cpp; src/server.cpp; src/client.cpp
+Libraries/Tools: System network libraries (Linux Sockets, Winsock2, etc.)
+Difficulty: Hard
+Status: Not Filled
+
+
+Details: Create an advance search engine using built-in LMDB database. More details will be discussed.
+Bounty Reward: $2000-$5000
```

Comment:
The bounty is okay, but I'd like to coordinate this work with whoever is up for it. If no one applies, I will do this.

Reply:
Okay, I hear you.